content/en/synthetics/ci.md

```diff
@@ -192,11 +192,66 @@ yarn add --dev @datadog/datadog-ci
 {{% /tab %}}
 {{< /tabs >}}
 
-### Adding and running tests
+### Setup the integration
 
-The configuration of synthetics test occurs in `**/*.synthetics.json` (configurable in the global configuration file). These files have a `tests` key, which contains an array of objects with the IDs of the tests to run and any configuration overrides for these tests.
+To setup your integration, Datadog API and application keys need to be configured. These keys can be defined in three different ways:
 
-**Example configuration file**:
+1. As environment variables:
+
+```
+export DATADOG_API_KEY="<API KEY>"
+export DATADOG_APP_KEY="<APPLICATION KEY>"
+```
+
+2. Passed to the CLI when running your tests: 
+
+```bash
+datadog-ci synthetics <command> --apiKey "<API KEY>" --appKey "<APPLICATION KEY>"
+```
+
+3. Or defined in a global configuration file:
+
+You can also create a JSON configuration file to specify more advanced options. The path to this global configuration file can be set using the flag `--config` [when launching your tests][#running-tests]. If the name of your global configuration file is set to `datadog-ci.json`, it defaults to it.
+
+* **apiKey**: The API key used to query the Datadog API.
+* **appKey**: The application key used to query the Datadog API.
+* **datadogSite**: The Datadog instance to which request is sent (choices are datadoghq.com or datadoghq.eu).
+* **files**: Glob pattern to detect synthetics tests config files.
+* **global**: Overrides of synthetics tests applied to all tests ([see below for description of each field][#configure-tests]).
+* **timeout**: Duration after which synthetics tests are considered failed (in milliseconds)
+
+**Example global configuration file**:
+
+```json
+{
+    "apiKey": "<DATADOG_API_KEY>",
+    "appKey": "<DATADOG_APPLICATION_KEY>",
+    "datadogSite": "datadoghq.com",
+    "files": "{,!(node_modules)/**/}*.synthetics.json",
+    "global": {
+        "allowInsecureCertificates": true,
+        "basicAuth": { "username": "test", "password": "test" },
+        "body": "{\"fakeContent\":true}",
+        "bodyType": "application/json",
+        "cookies": "name1=value1;name2=value2;",
+        "deviceIds": ["laptop_large"],
+        "followRedirects": true,
+        "headers": { "NEW_HEADER": "NEW VALUE" },
+            "locations": ["aws:us-west-1"],
+        "retry": { "count": 2, "interval": 300 },
+        "executionRule": "skipped",
+        "startUrl": "{{URL}}?static_hash={{STATIC_HASH}}",
+        "variables": { "titleVariable": "new title" }
+    },
+    "timeout": 120000
+}
+
+```
+### Configure tests
+
+The configuration of Synthetics tests occurs in `**/*.synthetics.json` (the path can be configured in the [global configuration file][#setup-the-integration]). These files have a `tests` key, which contains an array of objects with the IDs of the tests to run and any configuration overrides for these tests.
```

Comment:
IMO it's a bit unclear what is meant here. It's more than the configuration that "occurs" in those files, it's the very list of tests that will be run, along with some optional overriding configuration.

Reply:
True, clarifying that and resolving